thirdparty/kyaml/runfn/runfn.go

```diff
@@ -123,7 +136,15 @@ func (r RunFns) getNodesAndFilters() (
 		matchFilesGlob = append(matchFilesGlob, v1alpha2.KptFileName)
 	}
 	if r.Path != "" {
-		outputPkg = &kio.LocalPackageReadWriter{PackagePath: r.Path, MatchFilesGlob: matchFilesGlob}
+		piplineConfigFilter, err := r.getPipelineConfigFilterFn()
```

Comment:
Lets rename variable to `functionConfigFilter` to make it more intuitive.

Reply:
done